api/app/signals/apps/email_integrations/admin.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+
+from .models import EmailTemplate
+
+
+@admin.register(EmailTemplate)
+class EmailTemplate(admin.ModelAdmin):
```

Comment:
For now this is good enough, in the future we can add more help full tools and help texts

Reply:
Added helper text for body.

Reply:
Nice 👍